elements/erbium.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: erbium
```

Comment:
Looks great. We should capitalize Erbium here to keep is consistent with the fact.

Reply:
@vincent-nguyen-sys it should be changed now! Can you review?